File: server/src/main/java/ehealth/client/data_objects/LoginRequest.java
package ehealth.client.data_objects;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.*;


@EqualsAndHashCode
@AllArgsConstructor
@Getter
@Setter
@NoArgsConstructor
@ToString
public class LoginRequest {
    @JsonProperty("username")
    String username;
    @JsonProperty("password")
    String password;
}

File: client/CannaBe/CannaBe/DataObjects/Http/Requests/LoginRequest.cs
using Newtonsoft.Json;

namespace CannaBe
{
    class LoginRequest : Request
    {
        [JsonProperty("username")]
        string Username { get; set; }

        [JsonProperty("password")]
        string Password { get; set; }

        public LoginRequest(string username, string password)
        {
            Username = username;
            Password = password;
        }
    }
}
